loopd/swapclient_server.go

```diff
@@ -147,15 +147,20 @@ func (s *swapClientServer) Monitor(in *looprpc.MonitorRequest,
 
 	log.Infof("Monitor request received")
 
-	send := func(info loop.SwapInfo) error {
-		rpcSwap, err := s.marshallSwap(&info)
+	send := func(info *loop.SwapInfo) error {
```

Comment:
might have to do `swap := swap` in the loops where this is used now.

Reply:
I think we are ok, as we are passing in the loop variable as parameter instead of referencing it from the closure body.